spec/truffle/bigdecimal_spec.rb

```diff
@@ -84,6 +84,12 @@
       BigDecimal(@b, @a.precs[0]).to_s.should == "0.166666666666666667e3"
     end
 
+    it "BigDecimal(Rational) with bigger-than-double numerator" do
+      rational = 99999999999999999999/100r
+      rational.numerator.should > 2**64
+      BigDecimal(rational, 100).to_s.should == "0.99999999999999999999e18"
+    end
```

Comment:
Is there any reason to make this a Truffle-specific spec instead of a ruby/spec?

Reply:
I'll move the spec under `spec/ruby`, it passes on both Ruby 2.4 and 2.6.

Reply:
👍 . I'll move it

Reply:
I did it already on the internal branch, as well as moving the whole file to ruby/spec since there are TruffleRuby-specific calls there anymore.